.github/CODEOWNERS

```diff
@@ -0,0 +1,87 @@
+lib/vector-core/ @vectordotdev/core-team
+
+src/vrl/ @vectordotdev/processing-team
```

Comment:
Also `lib/vrl` and `lib/vector-vrl-functions`

Reply:
😓 `src/vrl` isn't a thing, should have been `lib`

Reply:
Resolved in c71059afb